views/article.html

```diff
@@ -80,6 +81,15 @@
 			}
 			</script>
 		</amp-analytics>
+		<amp-analytics type="krux" config="https://cdn.krxd.net/controltag/amp/KHUSeE3x.json">
+			<script type="application/json">
+			{
+				"extraUrlParams": {
+						"page.topics": "UK welfare reform,Minimum-Living Wage"
+					}
```

Comment:
we need to do something about this (although i don't know what Krux expects...the metadata comma separated?)